components/tools/Tags.tsx

```diff
@@ -0,0 +1,25 @@
+interface SelectTagsProps {
+  name?: string;
+  bgColor: string;
+  borderColor: string;
+};
+
+/**
+ * @description This component displays tags.
```

Comment:
Describe these tags more, on where it is used.

Reply:
Done